bin/migrate-oats-data/applications/submissions/data_insert/submap/soil_elements.py

```diff
@@ -1,21 +1,25 @@
 from enum import Enum
 
+
 class SoilAction(Enum):
-    RMV = 'REMOVE'
-    ADD = 'ADD'
-    
-alr_id = 'alr_appl_component_id'
-area = 'project_area'
-desc = 'material_desc'
-origin_desc = 'material_origin_desc'
-duration = 'project_duration'
-volume = 'volume'
-depth = 'depth'
+    RMV = "REMOVE"
+    ADD = "ADD"
+
+
+alr_id = "alr_appl_component_id"
+area = "project_area"
+desc = "material_desc"
+origin_desc = "material_origin_desc"
+duration = "project_duration"
+volume = "volume"
+depth = "depth"
+alt_measure = "alternative_measures_desc"
```

Comment:
Do you really need these constants? If yes, I would suggest wrapping them into an object or enum

Reply:
Done